cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1/types.go

```diff
@@ -45,6 +45,9 @@ type AppRepositorySpec struct {
 	// in the same namespace as the AppRepository and should be included
 	// automatically for matching images.
 	DockerRegistrySecrets []string `json:"dockerRegistrySecrets,omitempty"`
+	// In case of an OCI type, the list of repositories is needed
+	// as there is no API for the index
+	Repositories []string `json:"repositories,omitempty"`
```

Comment:
Just a thought, feel free to ignore: This object to which you're adding the `Repositories` is the spec of an `AppRepository`. Generally, an `AppRepository` as a repository of apps (whether that's a helm repository or something else in the future, such as, in this case, an OCI registry), so it initially hits some dissonance to then see a field called `Repositories`. Might be clearer to name it `OCIRepositories` so `appRepository.Spec.Repositories` becomes `appRepository.Spec.OCIRepositories`. See what you think.

Reply:
makes sense to me